docs/examples/driver_examples/QCoDeS Example with Yokogawa GS200 and Keithley 7510.ipynb

```diff
@@ -0,0 +1,1829 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "There are 0 readings in the buffer.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f'There are {buffer.last_index() - buffer.first_index()} readings in the buffer.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.3 Setup the tigger in"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, the falling edge will be used to trigger the measurement:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'FALL'"
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dmm.trigger_in_ext_edge()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'NONE'"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dmm.digitize_trigger()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We want an external trigger to trigger the measurement:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'EXT'"
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dmm.digitize_trigger('external')\n",
+    "dmm.digitize_trigger()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Check for errors"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "while True:\n",
+    "    smu_error = gs.system_errors()\n",
+    "    if 'No error' in smu_error:\n",
+    "        break\n",
+    "    print(smu_error)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "while True:\n",
+    "    dmm_error = dmm.system_errors()\n",
+    "    if 'No error' in dmm_error:\n",
+    "        break\n",
+    "    print(dmm_error)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Make the measurement"
```

Comment:
Please rewrite this section so that we only show how to make measurements with qcodes. We do not want to promote manual measurements with this driver that are different from every other driver

Reply:
The make measurement section has been rewritten.